demos/ag-alert-keyvault-advisor/azuredeploy.json

```diff
@@ -0,0 +1,64 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "alertName":{
+         "type":"String"
+      },
+      "applicationGatewayResourceGroup":{
+         "type":"String",
+         "metadata":{
+            "description":"Resource Group example /subscriptions/subs-id/resourceGroups/rg-name"
+         }
+      },
+      "actionGroupResourceID":{

```

Comment:
change this to prompt for the sub, rg and resource name - you can create the resourceId for the user in the template

Reply:
https://github.com/Azure/azure-quickstart-templates/pull/12205#discussion_r812779879